teamcity/project.go

```diff
@@ -75,6 +85,10 @@ func (r *projectResource) Create(ctx context.Context, req resource.CreateRequest
 		val := plan.Id.ValueString()
 		project.Id = &val
 	}
+	if !plan.ParentProjectId.IsUnknown() {
+		parentProjectId := plan.ParentProjectId.ValueString()
+		project.ParentProject = &client.Project{Id: &parentProjectId}
+	}
```

Comment:
should we fail the build if it's unknown (misspelled, etc.). Currently if it's unknown - it will become root by default. But build failure might be better, right?